### ISSUE TITLE
VirusTotal

### ISSUE THREAD

**Issue by MajorGeek** [Aetopia/NVIDIA-Driver-Downloader#1]
Way too many for anyone to consider it safe. Not saying it isn't, but it's blocked by everything including Google Chrome and Windows Defender - https://www.virustotal.com/gui/file/84c4685c1622bf4f0d207e7b5f6a3e5fc5f4b7d00288f8e84f255a51569ed596

**Comment by Aetopia** [comment]
Thank you for reporting!
I had also experienced this back in the past where a compiled/packed python script would receive many flags.

This occurs due to the following reasons:

1. Lack of file properties:
![image](https://user-images.githubusercontent.com/41850963/158925599-dd096bc7-40dd-4413-9f95-1af87576d7ea.png)
If there are no file properties, the executable will get flagged.

2. No Code signing, if the executable isn't code signed if might generate flags, but I found this to be less impactful.

I will push out a release this fixes this issue! Once again, thank you for reporting!
